### ISSUE TITLE
Privacy possum on Firefox seems outdated

### ISSUE THREAD

**Issue by re-hub** [cowlicks/privacypossum#273]
I could not info online or some place to ask. I beg your pardon if this id not the right place to ask. 

I  uninstalled pp because it seemed outdated. Yet here  I see recent commits. Is it intentionally not updated or sth?

Firefox 82 desktop
Firefox 83 mobile
2019.7.18

**Comment by zem99m** [comment]
Yes, it looks like the project owner has been making changes to the code during the past 12 months, which makes me think that an updated version will be released at some point - but the owner is also working on other projects, so Privacy Possum may not be a high priority. I am using Privacy Possum version 2019.7.18 in Firefox every day, and I don't see any problems with it. We would need the owner to post an update to know for sure.  Sorry I can't be of more help.

**Comment by Snaxeko** [comment]
> which makes me think that an updated version will be released at some point

It makes me think the project is discontinued, it has been removed from Chrome store and it should take just a few minutes to upload it again. I think that's the end for this superb extension.
